modules/HomePage/components/TaskItem/TaskItem.types.ts

```diff
@@ -0,0 +1,5 @@
+import { Task } from "../TaskContext/TaskContext";
+
+export interface TaskItemProps {
```

Comment:
```suggestion
export interface ITaskItemProps {
```

Same comment as above.